src/backend/tests/test_crawler_expunger.py

```diff
@@ -254,3 +257,72 @@ def test_expunger_for_record_with_mj_over_21(record_with_mj_over_21):
+                                                           reason="Never. Type ineligible charges are always time ineligible.",
+                                                           date_will_be_eligible=date.max())
+
+def test_expunger_for_record_with_mj_under_21_with_non_traffic_charge():
+    record = CrawlerFactory.create(
+        record=YoungDoe.SINGLE_CASE_RECORD,
+        cases={"CASEJD1": CaseDetails.CASE_MJ_AND_CIVIL_CONVICTION})
+
+    expunger_result = Expunger.run(record)
+
+    assert isinstance(record.charges[1].charge_type, CivilOffense)
+
+    assert expunger_result["CASEJD1-1"] == TimeEligibility(status=EligibilityStatus.ELIGIBLE,
+                                                           reason="Eligible now",
+                                                           date_will_be_eligible=date(1998, 3, 3)+relativedelta(years=1))
+
+    assert expunger_result["CASEJD1-2"] == TimeEligibility(status=EligibilityStatus.INELIGIBLE, reason="Never. Type ineligible charges are always time ineligible.",
+                                                           date_will_be_eligible=date.max())
+
+def test_expunger_for_record_with_mj_under_21_with_non_traffic_charge_not_subject_to_137226():
+    record = CrawlerFactory.create(
+        record=YoungDoe.SINGLE_CASE_RECORD,
+        cases={"CASEJD1": CaseDetails.CASE_MJ_AND_FUGITIVE_CONVICTION})
+
+    expunger_result = Expunger.run(record)
+
+    assert isinstance(record.charges[0].charge_type, MarijuanaUnder21)
+    assert isinstance(record.charges[1].charge_type, CivilOffense)
+
+    assert expunger_result["CASEJD1-1"] == TimeEligibility(status=EligibilityStatus.ELIGIBLE,
+                                                           reason="Eligible now",
+                                                           date_will_be_eligible=date(1998, 3, 3)+relativedelta(years=3))
+
+    assert expunger_result["CASEJD1-2"] == TimeEligibility(status=EligibilityStatus.INELIGIBLE, reason="Never. Type ineligible charges are always time ineligible.",
+                                                           date_will_be_eligible=date.max())
+
+def test_expunger_for_record_with_mj_under_21_blocked_by_blocking_conviction():
+    record = CrawlerFactory.create(
+        record=YoungDoe.SINGLE_CASE_RECORD,
+        cases={"CASEJD1": CaseDetails.CASE_MJ_AND_FELONY_CONVICTION})
+    expunger_result = Expunger.run(record)
+
+    mj_charge = record.charges[0]
+    felony_charge = record.charges[1]
+
+    assert isinstance(mj_charge.charge_type, MarijuanaUnder21)
+
+    assert expunger_result["CASEJD1-2"] == TimeEligibility(status=EligibilityStatus.INELIGIBLE, reason="Never. Type ineligible charges are always time ineligible.",
+                                                           date_will_be_eligible=date.max())
+    assert expunger_result["CASEJD1-1"] == TimeEligibility(status=EligibilityStatus.ELIGIBLE,
```

Comment:
Nitpick but I prefer to be consistent about the ordering: assert for CASEJD1-1 before CASEJD1-2.